### ISSUE TITLE
Is doc for foolbox outdated ?

### ISSUE THREAD

**Issue by lingjunz** [bethgelab/foolbox#331]
I found the [doc](https://foolbox.readthedocs.io/en) for foolbox mismatch the actual api.
```
model = KerasModel(orig_kmodel,(0,1), channel_axis=channel,predicts = 'logits')
model.forward(batch_x/255.0)

(Failed)output: 'KerasModel' object has no attribute 'farword'
```
```
model = KerasModel(orig_kmodel,(0,1), channel_axis=channel,predicts = 'logits')
model.batch_predictions(batch_x/255.0)

(This works.)
```

Is there a new doc for foolbox?
Thanks.

**Comment by jonasrauber** [comment]
Sorry, actually the doc is "too new" … it's built from the master branch and therefore corresponds to the Foolbox 2.0 beta version.

**Comment by jonasrauber** [comment]
I fixed it now to show the docs for 1.8 by default. One can switch to the 2.0b0 docs by selecting _latest_ in the lower left corner of the docs.
